README.md

```diff
@@ -45,6 +45,22 @@ If you haven't, create a superuser with the [Django createsuperuser command](htt
 - Create a new drip.
 - Select and delete drips.
 
+### Support SMS(Optional)
+Since sending an sms can be done via several APIs, different ways and different conditions suitable to each project, we've decide to decoupled the code for sending sms using signals.
+Therefore,
+- In your signals.py use the post_drip signal receiver to handle the logic for sending the sms
+```python
+from django.dispatch import receiver
+from drip.signals import post_drip
+
+@receiver(post_drip)
+def handle_drip_sms(sender, **kwargs):
+    drip = kwargs['drip']
+    user = kwargs['user']
+    # Get the message from the drip and the number from the user and handle the SMS message logic.
+```
+- The sms text field supports user object injection, so you can use placeholders for the user data `eg: This sms for {{user.first_name}}` 
+
```

Comment:
should we add instructions on how to support translation?

Reply:
Yeah I know I haven't touch the readme yet. I will get back to it